data/sql/updates/pending_db_world/2016_09_02_01.sql

```diff
@@ -0,0 +1,11 @@
+ALTER TABLE version_db_auth CHANGE COLUMN 2016_09_02_01 bit;
```

Comment:
Please follow the guide to create sql PR, it's very simple now: https://github.com/azerothcore/azerothcore-wotlk/wiki/Dealing-with-SQL-files#1-creating-a-versioned-pending-sql-file--recommended-

just 4 steps and you don't need to insert any ALTER TABLE manually